src/util/colors.js

```diff
@@ -0,0 +1,27 @@
+// @ts-check
+/** Convert hex color to rgb
+ * @param {string} hex
+ * @returns {(object|null)}
+ * @see https://stackoverflow.com/a/5624139/3367568
+ */
+export function hexToRgb(hex) {
```

Comment:
As suggested in the last section (Update) of the answer in stack overflow, we can implement that function in order to handle the shorthand form of hex colors (e.g. "03F").

Reply:
Done